_data/contributors.yml

```diff
@@ -195,6 +195,26 @@
       report: |
         [URL](https://indico.cern.ch/event/1066812/contributions/4495290/attachments/2302817/3917355/Iris%20HeP.pdf"){:target="_blank"}
 
+- name: Sunho Kim
```

Comment:
Can you move your profile entry after the one of Ioana?

Reply:
Done